Improve explanations in getting started tutorial (#9299)
wagtail/wagtail: docs/getting_started/tutorial.md

Wagtail supports Python 3.7, 3.8, 3.9 and 3.10.

> To check the current version of python currently installed on your machine, run the command bellow:

Reviewer: ```suggestion
To check whether you have an appropriate version of Python 3:
```

@Dev-Yusuf I’m not necessarily opposed to changing this but I think it’s important we keep the wording for this relatively short.

```doscon
> python3 -m venv mysite\env

> > mysite\env\Scripts\activate.bat
> 
> ```
> Or 
> 
> ```doscon
> > python3 -m venv "name-of-your-virtual-environment"
> > "name-of-your-virtual-environment"\Scripts\activate
> 

Reviewer: Even if we add more variations I think it’s important we keep the same example.

```suggestion
> mysite\env\Scripts\activate.bat

Or:

> mysite\env\Scripts\activate
```

Because the folder `mysite` was already created by `venv`, run `wagtail start` with an additional argument to specify the destination directory:

```console

> $ wagtail start mysite

Reviewer: ```suggestion
$ wagtail start mysite mysite
```

@Dev-Yusuf this doesn’t seem correct? The line above explains why we give this `mysite` twice.

$ pip install -r requirements.txt
```

> the `requirements.txt` file contains all the dependencies needed
> in order to run the project.

Reviewer: @Dev-Yusuf happy for us to consider changes to this but I felt like the existing content was useful. Perhaps we could add your content in addition, rather than instead?

```suggestion
This ensures that you have the relevant versions of Wagtail, Django, and any other dependencies for the project you have just created.
The `requirements.txt` file contains all the dependencies needed in order to run the project.
```

{% endblock %}
```

> `base.html` refers to a parent template and must always be the first template tag used in a template. Extending from this template saves you from rewriting codes and allows pages across your app to share a similar frame (by using block tags in the child template, you are able to override specific content within the parent template).

Reviewer: ```suggestion
`base.html` refers to a parent template and must always be the first template tag used in a template. Extending from this template saves you from rewriting code and allows pages across your app to share a similar frame (by using block tags in the child template, you are able to override specific content within the parent template).
```
